### DISCUSSION TITLE
What should I do if my server is filtered? اگه سرورم فیلتر شد چیکار کنم؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#123] [Q&A سوالات رایج]
اگه سرورم فیلتر شد چیکار کنم؟

**Comment by Shahzad55** [comment]
ساب دامین تعریف کن هم آپی ورژن 4 هم با IPV6 باAAAA Type  پاراکسی روشن کلودفلر توی پنل هم میتونی IP تمیز رو حالت CDN پیدا کنی قرار بدی
